zk-rs/src/account.rs

```diff
@@ -0,0 +1,319 @@
+use ark_bn254::Fr;
+use crypto_box::SecretKey;
+use num_bigint::BigUint;
+use wasm_bindgen::prelude::wasm_bindgen;
+
+use crate::{
+    blake2::{blake2_hash, blake2_hash_str},
+    poseidon::poseidon_hash,
+};
+
+#[wasm_bindgen]
+pub struct Account {
+    // solana_public_key: Vec<u8>,
+    // seed: Vec<u8>,
+    public_key: Vec<u8>,
+    private_key: Vec<u8>,
+    encryption_public_key: Vec<u8>,
+    encryption_private_key: Vec<u8>,
+    aes_secret: String,
+    hashing_secret: String,
+}
+
+#[wasm_bindgen]
+impl Account {
+    #[wasm_bindgen(constructor)]
+    pub fn new(seed: String) -> Self {
+        // if seed.len() < 32 {
```

Comment:
We discussed to use a seed length of 31 bytes for now right?

Reply:
another way would be to use a longer seed and just hash it to reduce size.

Reply:
I'm reducing the size for shieldedPrivateKey [here](https://github.com/Lightprotocol/light-protocol/pull/366/files#diff-89daba50c0798a99c66343b925c3ddc10390f9ee494030b3e5d02023ff2b7243R433), could you please check if this is correct?